README.rst

```diff
@@ -138,12 +138,13 @@ Other publications:
 Copyright and license
 ---------------------
 
-Charliecloud is copyright © 2014–2023 Triad National Security, LLC and others.
+Charliecloud is copyright © Charliecloud a Series of LF Projects, LLC and
+others.
```

Comment:
This is probably wrong. The LF project doesn't own the copyright to any of the code in Charliecloud -- Triad (and other contributors) still do. You don't really need a copyright statement in the README, so I'd just take it out.